_random.py

```diff
@@ -0,0 +1,53 @@
+import random
+
+def int(start, end, count=1):
+    """ 
+    A function to return 'n' random integers between a given range 'Start' to 'End'
```

Comment:
These methods are so simple that we don't need documentation comments, you can remove them.